src/Public/OpenApi.ps1

```diff
@@ -1778,6 +1862,16 @@ The email address of the contact person/organization. MUST be in the format of a
 .PARAMETER ContactUrl
 The URL pointing to the contact information. MUST be in the format of a URL.
 
+.PARAMETER SpecTag
+A string representing the unique tag for the API specification.
+This tag helps in distinguishing between different versions or types of API specifications within the application.
+Use this tag to reference the specific API documentation, schema, or version that your function interacts with.
+
+.PARAMETER SpecTag
+A string representing the unique tag for the API specification.
+This tag helps in distinguishing between different versions or types of API specifications within the application.
+Use this tag to reference the specific API documentation, schema, or version that your function interacts with.
+
```

Comment:
Copy-pasta duplication 😉

Reply:
The plan is to create multiple openapi documents. The name, I agree, is not the best. I leave it to you to find a better one ☺️ 
The name comes from a ChatGPT suggestion 😅 because I had no idea what to call it.
Anyway, today I'm going to commit a fully workable version with “spectag” 
Feedback are welcome